### ISSUE TITLE
LightningCSS does not work in dev mode

### ISSUE THREAD

**Issue by predaytor** [HorusGoul/vite-plugin-stylex#59]
Stackblitz (https://stackblitz.com/edit/remix-run-remix-vneo6e?file=app%2Froutes%2F_index.tsx)

Post-processing with LightningCSS does not work in dev mode. The production build works as expected.
```tsx
const styles = stylex.create({
  x: {    
     color: 'lch(from pink calc(l + 10%) c h)',  
  },
});
```
css output:
```css
color: lch(from pink calc(l + 10%) c h);
```
should be:
```css
color: lch(from pink calc(l + 10%) c h);
color: lab(93.7873% 24.4367 3.75943);
```
Notice the defined CSS in the regular imported stylesheet (in our case the main import `/styles/index.css`), the post-processing is working properly.

**Comment by HorusGoul** [comment]
Could you try `vite-plugin-stylex@0.9.0-next.0` to see if the plugin works better with Lightning CSS?

See the changelog just in case the breaking changes affect your project: https://github.com/HorusGoul/vite-plugin-stylex/blob/waitForRequestsIdle-refactor/packages/vite-plugin-stylex/CHANGELOG.md

**Comment by predaytor** [comment]
@HorusGoul same behavior, dev mode doesn't apply lightningcss post-processing.

**Comment by HorusGoul** [comment]
@predaytor Just released `0.9.0-next.1`, this one works as expected in the Stackblitz you shared (StyleX generated style and the .primary styles have the same result)

![image](https://github.com/HorusGoul/vite-plugin-stylex/assets/6759612/54be5c66-d084-4169-800d-c4fd085d5ab4)

I believe any future errors related to Lightning might come from either Vite or Lightning themselves, there's not a lot more I can do from this plugin :thinking:

**Comment by predaytor** [comment]
thx!
